scenes/MainScene.tscn

```diff
@@ -14,6 +15,8 @@ tonemap_mode = 2
 [node name="GameWorld" type="Spatial"]
 script = ExtResource( 1 )
 
+[node name="Root" parent="." instance=ExtResource( 4 )]
+
 [node name="Sun" type="DirectionalLight" parent="."]
 transform = Transform( 1, 0, 0, 0, 0.707107, 0.707107, 0, -0.707107, 0.707107, 0, 14, 0 )
```

Comment:
So I think I understand - the sun is a giant directional light and you change its rotation to modulate the brightness?

Reply:
Well, to change the *angle*, which does influence the brightness, yes!